include/cantera/kinetics/Arrhenius.h

```diff
@@ -268,6 +268,15 @@ class TwoTempPlasmaRate final : public ArrheniusBase
                               * shared_data.recipTe * shared_data.recipT);
     }
 
+    //! Evaluate derivative of reaction rate with respect to temperature
+    //! divided by reaction rate
+    /*!
+     *  @param shared_data  data shared by all reactions of a given type
+     */
+    virtual double ddTScaledFromStruct(const ReactionData& shared_data) const {
+        return ((m_Ea_R - m_EE_R) * shared_data.recipT) * shared_data.recipT;
+    }
+
```

Comment:
This was already implemented in #1184 ... could you rebase as I believe there are now some merge conflicts?

Reply:
Thanks! I should have realized it!

Reply:
#1184 introduced several simplifications to cut down on redundant code, although none of the fundamentals have changed.